admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/RegistrationCenterServiceImpl.java

```diff
@@ -1115,40 +1115,16 @@ public RegistrationCenterExtnDto createRegistrationCenter(RegCenterPostReqDto re
 	 * @return
 	 */
 	@Transactional
-	private WorkingNonWorkingDaysDto setResponseDtoWorkingNonWorking(RegistrationCenter registrationCenter) {
+	private Map<String, Boolean> setResponseDtoWorkingNonWorking(RegistrationCenter registrationCenter) {
 		List<RegWorkingNonWorking> workingNonWorkingDays = regWorkingNonWorkingRepo
-				.findByRegCenterId(registrationCenter.getId());		
-		WorkingNonWorkingDaysDto workDays = new WorkingNonWorkingDaysDto();
+				.findByRegCenterId(registrationCenter.getId());
+		Map<String, Boolean> workingNonWorkingDay = new HashedMap<String, Boolean>();
```

Comment:
just use diamond operator in the right part of init don't add classes new  HashedMap<>()

Reply:
taken care